.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -21,14 +21,23 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
 import model.CIBuildModel
 import model.StageNames
 
-val killAllGradleProcesses = """
+val killAllGradleProcessesLinux = """
```

Comment:
Is this for Linux or UNIX/POSIX (meaning it should work on macOS too)?

Reply:
I see we use `UnixLike` vs `Windows` elsewhere.

Reply:
done.